Data/IntMap/Internal.hs

```diff
@@ -2370,30 +3097,30 @@ showTreeWith hang wide t
   | otherwise = (showsTree wide [] [] t) ""
 
```

Comment:
Can we add some deprecation pragmas to eventually move these debugging functions out of `Data.IntMap` and into a debugging module?